pkg/cli/verifier/verifier.go

```diff
@@ -35,9 +36,13 @@ type Verifier interface {
 	Run() Result
 }
 
+func marker(count int) string {
+	return "[" + strings.Repeat("+", count) + "]"
+}
+
 // Print prints the Result
-func Print(result Result, w io.Writer) {
-	fmt.Fprintf(w, "[+] Context: %s\n", result.Context)
+func Print(result Result, w io.Writer, markerDepth int) {
```

Comment:
I wonder if indentation would be more clear to show there's a hierarchy to these.

Reply:
In my opinion, using indentation quickly gets out of hand when the nesting depth increases. I prefer to keep this as is.